task/rhacs-deployment-check/3.71/samples/pipeline.yaml

```diff
@@ -26,6 +28,25 @@ spec:
       description: image to be build from the code
 
   tasks:
+    - name: enable-workspace-write
+      taskRef:
+        name: enable-write-on-workspace
```

Comment:
Why do we need it?

Reply:
The workspace seems to be created with owner `root:root` and mode `0755`, which breaks the fetch-repository/git-clone step.
The documentation of the task (https://github.com/tektoncd/catalog/tree/main/task/git-clone/0.9) suggests the addition of a security context to run as non-root user, now this addition would be a change to thePipelineRun sample as well, and the various steps and associated container images run with different users, which may break if not run with the appropriate security context.
In the end, I'll remove this step from the sample. IMHO, the principle is more to show how to configure the task itself.